init/modules/EESSI/2023.06.lua

```diff
@@ -0,0 +1,67 @@
+help([[
+Description
+===========
+The European Environment for Scientific Software Installations (EESSI, pronounced as easy) is a collaboration between different European partners in HPC community.The goal of this project is to build a common stack of scientific software installations for HPC systems and beyond, including laptops, personal workstations and cloud infrastructure. 
+
+More information
+================
+ - URL: https://www.eessi.io/docs/
+]])
+whatis("Description: The European Environment for Scientific Software Installations (EESSI, pronounced as easy) is a collaboration between different European partners in HPC community. The goal of this project is to build a common stack of scientific software installations for HPC systems and beyond, including laptops, personal workstations and cloud infrastructure.")
+whatis("URL: https://www.eessi.io/docs/:")
```

Comment:
Just curious, is the last `:` part of the URL or part of the `whatis`-syntax? Maybe it should be

```suggestion
whatis("URL: https://www.eessi.io/docs/")
```